services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -274,7 +274,7 @@ export const SubmissionSummary = (): React.ReactElement => {
             >
                 {renderStatusAlerts()}
 
-                {documentError && (
+                {!documentError && (
```

Comment:
Is this intentional? I think we would want it to show when there's an error

Reply:
Oops that was for making the banner show. Good catch